charts/vela-core/templates/velaConfig.yaml

```diff
@@ -9,5 +9,14 @@ data:
       "repo": "jetstack",
       "urL": "https://charts.jetstack.io",
       "name": "cert-manager",
-      "version": "v1.0.0"
-    },
\ No newline at end of file
+      "namespace": "cert-manager",
+      "version": "1.0.0"
+    }
+  ingresses.networking.k8s.io: |
```

Comment:
please add some readme about how this config get to work

Reply:
I will after the whole e2e works, it's still a mess